examples/bri-3/src/bri/zeroKnowledgeProof/agents/ccsmAnchorHash.agent.ts

```diff
@@ -0,0 +1,49 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { INVALID_ANCHOR_HASH_INPUT } from '../api/err.messages';
+import { v4 as uuidv4 } from 'uuid';
+import { CCSMAnchorHash } from '../models/ccsmAnchorHash';
+
+@Injectable()
+export class CCSMAnchorHashAgent {
+  public throwErrorIfCCSMAnchorHashInputInvalid(
+    inputForProofVerification: string,
```

Comment:
@biscuitdey should this not be an array?

Reply:
For the current iteration, it is kept as a string. Will modify it in future PR. Here is the [document](https://docs.google.com/document/d/1mkO5oou0TWk6kVHbTB01dyj7wuNGTfR9w6PAxSgXdGI/edit?usp=sharing) that details future implementations.